objects/obj_ini/Create_0.gml

```diff
@@ -77,7 +79,9 @@ millenium=0;
 
 if (instance_exists(obj_creation)) then custom=obj_creation.custom;
 
+global.unit_init = true;
```

Comment:
shouldn't this be in reverse? i.e. `unit_init = true` should mean that units have been initialized?

Reply:
think i since made this redundant shall check and if so remove